xtractmime/_utils.py

```diff
@@ -0,0 +1,252 @@
+from struct import unpack
+from typing import Tuple, Union
+
+from xtractmime import _is_match_mime_pattern
+from xtractmime._patterns import (
+    IMAGE_PATTERNS,
+    AUDIO_VIDEO_PATTERNS,
+    FONT_PATTERNS,
+    ARCHIVE_PATTERNS,
+)
+
+sample_rates = [44100, 48000, 32000]
+mp25_rates = [
+    0,
+    8000,
+    16000,
+    24000,
+    32000,
+    40000,
+    48000,
+    56000,
+    64000,
+    80000,
+    96000,
+    112000,
+    128000,
+    144000,
+    160000,
+]
+mp3_rates = [
+    0,
+    32000,
+    40000,
+    48000,
+    56000,
+    64000,
+    80000,
+    96000,
+    112000,
+    128000,
+    160000,
+    192000,
+    224000,
+    256000,
+    320000,
+]
+
+
+def _is_mp4_signature(input_bytes: bytes) -> bool:
```

Comment:
Now that it’s `_utils`, you no longer need to keep these function names private as well.

```suggestion
def is_mp4_signature(input_bytes: bytes) -> bool:
```